README.md

```diff
@@ -168,8 +168,47 @@ const BtnClassName = css`
   border-radius: 4px;
 `;
 
+// vanilla JS
 const btn = document.querySelector("#btn");
-btn.classList.add(BtnClassName);
+// BtnClassName() -> 'g016232'
+btn.classList.add(BtnClassName());
+
+// JSX
+// BtnClassName() -> 'g016232'
+const App => <button className={BtnClassName()}>click</button>
+```
+
+#### Different ways of customizing `css`
+##### Tagged templates functions
+```js
+import { css } from "goober";
+
+const BtnClassName = css`
+  border-radius: ${props=>props.size}px;
+`;
+
+// vanilla JS
+// BtnClassName({size:20}) -> g016360
+const btn = document.querySelector("#btn");
+btn.classList.add(BtnClassName({size:20}));
+
+// JSX
+// BtnClassName({size:20}) -> g016360
+const App => <button className={BtnClassName({size:20})}>click</button>
```

Comment:
I guess there's a set of braces missing?
```suggestion
const App = () => <button className={BtnClassName({size:20})}>click</button>
```

Reply:
aaah right! thx!